boards/FSM/Src/main.c

```diff
@@ -136,6 +136,11 @@ int main(void)
     __HAL_DBGMCU_FREEZE_IWDG();
     Io_SharedHardFaultHandler_Init();
 
+    Io_FlowMeters_Init(&htim4);
```

Comment:
Flow meters are initialized in user code begin 1 for now

Reply:
Also noticed that I accidentally added the `App_FlowMeter_Tick` to the 1kHz task not the 1Hz task. I've fixed it and updated the code but its not showing up on this pr. :thinking: